include/osg/GLDefines

```diff
@@ -566,6 +566,7 @@ typedef char GLchar;
 #define GL_UNSIGNED_INT64_VEC2_ARB 0x8FF5
 #define GL_UNSIGNED_INT64_VEC3_ARB 0x8FF6
 #define GL_UNSIGNED_INT64_VEC4_ARB 0x8FF7
+
```

Comment:
Did include/osg/GLBeginEndAdapter get removed completely and is no longer used by the particle system?